docs/docs/performance.md

```diff
@@ -128,21 +134,49 @@ Benchmarks on an Macbook Pro M1 Max (2021).
+|        Stdev         |                       31.9 ms                       |                      75.2 ms                      |                          168 ms                           |                   58.6 ms                   |
+| Comparing to snarkjs |                        ~1.9x                        |                       ~1.5x                       |           <font color="FFB546">**~3.3**</font>            |                      -                      |
 
 <iframe width="472" height="263" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQOuS5abdzvh6znXORvSo7M85ubmDpSmE3C1Zs_l56wd25lMK4FZPEWOaCp7WrOlIjc3jEcWa0lfiy9/pubchart?oid=222721960&amp;format=interactive"></iframe>
 
-| RSA | [Tachyon](https://github.com/kroma-network/tachyon) | [rapidsnark](https://github.com/iden3/rapidsnark) | [ark-works](https://github.com/arkworks-rs/circom-compat) | [snarkjs](https://github.com/iden3/snarkjs) |
-| :--: | :--: | :--: | :--: | :--: | 
-| Average | 2307 ms | 2560 ms | 2530 ms | 5504 ms |
-| Stdev | 18.7 ms | 21.3 ms | 266.1 ms | 69.3 ms |
-| Comparing to snarkjs | <font color="FFB546">**~2.3**</font> | ~2.1x | ~2.1x | - |
+|         RSA          | [Tachyon](https://github.com/kroma-network/tachyon) | [rapidsnark](https://github.com/iden3/rapidsnark) | [ark-works](https://github.com/arkworks-rs/circom-compat) | [snarkjs](https://github.com/iden3/snarkjs) |
+|:--------------------:|:---------------------------------------------------:|:-------------------------------------------------:|:---------------------------------------------------------:|:-------------------------------------------:| 
+|       Average        |                       2307 ms                       |                      2560 ms                      |                          2530 ms                          |                   5504 ms                   |
+|        Stdev         |                       18.7 ms                       |                      21.3 ms                      |                         266.1 ms                          |                   69.3 ms                   |
+| Comparing to snarkjs |        <font color="FFB546">**~2.3**</font>         |                       ~2.1x                       |                           ~2.1x                           |                      -                      |
 
 <iframe width="472" height="279" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQOuS5abdzvh6znXORvSo7M85ubmDpSmE3C1Zs_l56wd25lMK4FZPEWOaCp7WrOlIjc3jEcWa0lfiy9/pubchart?oid=1443184132&amp;format=interactive"></iframe>
 </details>
 
-**Details:** [Spreadsheet of Circom benchmark (macOS)](https://docs.google.com/spreadsheets/d/1irKg_TOP-yXms8igwCN_3OjVrtFe5gTHkuF0RbrVuho/edit?usp=sharing)
+**Details:
+** [Spreadsheet of Circom benchmark (macOS)](https://docs.google.com/spreadsheets/d/1irKg_TOP-yXms8igwCN_3OjVrtFe5gTHkuF0RbrVuho/edit?usp=sharing)
```

Comment:
```suggestion
**Details:** [Spreadsheet of Circom benchmark (macOS)](https://docs.google.com/spreadsheets/d/1irKg_TOP-yXms8igwCN_3OjVrtFe5gTHkuF0RbrVuho/edit?usp=sharing)
```

Reply:
I sware I fixed it. Sorry for the autocorrection doing it again!